onnx/onnx.in.proto

```diff
@@ -379,16 +383,24 @@ message ModelProto {
   // all stored `TrainingInfoProto.initialization`s with the associated
   // `TrainingInfoProto.initialization_binding`s.
   //
-  // To use GraphCall and Gradient operators to generate gradient of trainable
-  // tensors, the user should put all trainable tensors to three places.
-  //  - inference graph's input list.
-  //  - inference graph's initializer list.
-  //  - update_binding(s) of the stored TrainingInfoProto.
-  // In addition, when using GraphCall, the user should also explicitly
-  // specify all trainable tensors in the input list of the GraphCall operator.
-  //
   // If this field is empty, the training behavior of the model is undefined.
-  repeated TrainingInfoProto training_info = 21;
+  repeated TrainingInfoProto training_info = 20;
+
+  // The following denote mutable global variables visible in ModelProto.graph as all as
+  // all graphs in TrainingInfoProto. These are initialized to the values specified
+  // in the initializer below, and then updated by initializer-bindings and
+  // update-bindings.
+  //
+  // If an input or initializer in a graph has the same name as a global variable,
+  // then the local variable in the graph will hide the global variable.
+  //
+  // This field usually stores trainable tensors of the model, because trainable
+  // tensors are used by both inference graph and training algorithm graph.
+  repeated TensorProto global_initializer = 30;
```

Comment:
```suggestion
  repeated TensorProto global_mutable_initializer = 30;
```
maybe?

Reply:
Let's make it string. All tensors are in global_initializer, the name of the mutable subset are in global_mutable_initializer.

Reply:
Ok.